string req
---
string hintStatement
bool result
